docs/app/documentation/component-docs/dropdown/examples/dropdown-icons-example.component.html

```diff
@@ -1,12 +1,42 @@
 <fd-popover>
     <fd-dropdown-control [glyph]="'filter'">Dropdown</fd-dropdown-control>
-    <fd-popover-body *ngIf="menu1 && menu1.length">
-        <fd-menu>
+    <fd-popover-body *ngIf="values && values.length">
+        <fd-menu fd-menu-addon>
             <ul fd-menu-list>
-                <li *ngFor="let option of menu1">
-                    <a fd-menu-item [attr.href]="option.url">{{option.text}}</a>
-                </li>
+                <li *ngFor="let option of values">
+                    <div fd-menu-item-addon>
+                        <fd-icon size="l" [glyph]="option.icon" class="fd-template-icon"></fd-icon>
+                    </div>
+                    <a href="" fd-menu-item>{{option.name}}</a>
+
             </ul>
+
         </fd-menu>
     </fd-popover-body>
 </fd-popover>
+
+<!-- Additional styles -->
+<style>
+    .fd-template-container-div {
+        display: flex;
+        align-items: center;
+        cursor: pointer;
+    }
+
+    .fd-template-container-div:hover {
+        background-color: var(--fd-color-background-hover);
+    }
+
+    .fd-menu__item {
+        display: inline-block;
+        cursor: pointer;
+        padding: 10px;
+        background-color: var(--fd-menu-item-background-color);
+        color: var(--fd-menu-item-color);
+        font-weight: 400;
+    }
+
+    .fd-template-icon {
+        padding-left: 10px;
+    }
+</style>
```

Comment:
Please remove styles, it's not needed since we have these directives.

Reply:
@JKMarkowski i have removed the css please approve the request